src/modules/interest-rates/entities/interest-rate.entity.ts

```diff
@@ -0,0 +1,58 @@
+import { ClassSerializerInterceptor, UseInterceptors } from '@nestjs/common';
+import { ApiProperty } from '@nestjs/swagger';
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({
+  name: 'DWR_INT_RATE',
+  database: 'cedar',
```

Comment:
database name is not required and it works better without if we use one DB for testing.

Reply:
I've tried to remove it but for some reason, it breaks the server

Reply:
updated